domain-ee/ee-max-cfi-app/src/python_src/util/data/Max Ratings DC Lookup Table v2.csv

```diff
@@ -0,0 +1,60 @@
+Diagnostic Code,Rated Issue Name,Max Rating (%),Body System,Category,Subcategory,CFR_Ref
+7200,"Soft tissue injury of the mouth, other than tongue or lips:",90,The Digestive System,,,38 CFR 4.114
+7201,"Lips, injuries of.",90,The Digestive System,,,38 CFR 4.114
+7202,"Tongue, loss of whole or part.",100,The Digestive System,,,38 CFR 4.114
+7203,"Esophagus, stricture of.",80,The Digestive System,,,38 CFR 4.114
+7204,Esophageal motility disorder:,80,The Digestive System,,,38 CFR 4.114
+7205,"Esophagus, diverticulum of, acquired.",80,The Digestive System,,,38 CFR 4.114
+7206,"Gastroesophageal reflux disease:	",80,The Digestive System,,,38 CFR 4.114
+7207,Barrett’s esophagus:,80,The Digestive System,,,38 CFR 4.114
+7301,"Peritoneum, adhesions of, due to surgery, trauma, disease, or infection:",80,The Digestive System,,,38 CFR 4.114
+7303,"Chronic complications of upper gastrointestinal surgery:	",80,The Digestive System,,,38 CFR 4.114
```

Comment:
This almost certainly doesn't really matter, but I noticed that the rated issue names in v2 often end with a colon, and occasionally a colon followed by some number of spaces. In v2, they end with a period or nothing at all. 🤷